.mvn/develocity.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
@ribafish  Please provide license header for this new file, you can use an existing .xml file as an example

Reply:
Added in [13d4904](https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/665/commits/13d49040cda733e62564f7da87538cc371e5b6e3)